projects/observability/src/shared/components/explore-query-editor/explore-query-editor.component.ts

```diff
@@ -137,6 +162,10 @@ export class ExploreQueryEditorComponent implements OnChanges, OnInit {
     this.visualizationBuilder.interval(interval === 'NONE' ? undefined : interval);
   }
 
+  public setOrderBy(orderBy: GraphQlOrderBy): void {
```

Comment:
nit: unnecessary wrapper, it's identical to `updateOrderByExpression` above.

Reply:
removed